src/main/java/frc/robot/subsystems/Swerve/DrivebaseSubsystem.java

```diff
@@ -184,15 +190,29 @@ public Rotation2d getGyroscopeRotation() {
     public void periodic() {
 
         swerveOdometry.update(getGyroscopeRotation(), getModulePositions());
-
+        System.out.println(ll.getRobotPose());
         Optional<EstimatedRobotPose> result = pcw.getEstimatedGlobalPose(swerveOdometry.getEstimatedPosition());
-        /* 
+
         if (result.isPresent()) {
             EstimatedRobotPose camPose = result.get();
             swerveOdometry.addVisionMeasurement(
                     camPose.estimatedPose.toPose2d(),
                     camPose.timestampSeconds);
-        }*/
+        }
+        LimelightResults llresults = LimelightHelpers.getLatestResults("limelight");
+
+        NetworkTable table = NetworkTableInstance.getDefault().getTable("limelight");
+        NetworkTableEntry tl = table.getEntry("tl");
+        NetworkTableEntry cl = table.getEntry("cl");
+        NetworkTableEntry tv = table.getEntry("tv");
+
+        double totalLatency = cl.getDouble(1) + tl.getDouble(1);
+        Pose2d please_work = llresults.targetingResults.getBotPose2d_wpiBlue();
+        double hasTarget = tv.getDouble(0);
+
+        double im_crying = Timer.getFPGATimestamp() - (totalLatency / 1000.0);
+        if (hasTarget == 1)
```

Comment:
Does this `1` mean "I only see a single april tag"? Would be good to make a note in code or in the PR/commit why that'd be the limit right now.